Client/src/components/contanier/staff/Student/AddStudent/index.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react'
+import LabeledInput from './../../../../common/LabeledInput'
+import Button from './../../../../common/Button';
+import { addStudentSchema } from './../../../../../helpers/validation-schema'
+import './index.css';
+let valid;
+
+export default class AddStudent extends Component {
+  state = {
+    firstname: '',
+    address: '',
+    level: 0,
+    phonenumber: '',
+    lastname: '',
+    mobilenumber: '',
+    password: '',
+    Error: {}
+  }
+  handleInput = ({ target: { value, name } }) => this.setState({ [name]: value.trim() });
+
+  handleClick = (e) => {
+    e.preventDefault();
+    this.setState({ Error: '' })
```

Comment:
@Fatmasiam Just to be consistent and always have Error be an object, you could do
`this.setState({ Error: {} })`